cwe_api/Dockerfile

```
@@ -3,10 +3,26 @@ FROM python:3.12-slim
 WORKDIR /app
 
 COPY . .
+ARG FILE_ID
 
-RUN pip install torch torchvision torchaudio --index-url https://download.pytorch.org/whl/cpu
+RUN set -e \
```

Hadolint

[warning] Pin versions in apt get install. Instead of `apt-get install <package>` use `apt-get install <package>=<version>`

(DL3008)